uninstall.php

```diff
@@ -42,3 +42,4 @@
 // Drop our custom tables.
 $wpdb->query( "DROP TABLE IF EXISTS {$wpdb->prefix}tec_order_modifiers" );
 $wpdb->query( "DROP TABLE IF EXISTS {$wpdb->prefix}tec_order_modifiers_meta" );
+$wpdb->query( "DROP TABLE IF EXISTS {$wpdb->prefix}tec_order_modifier_relationships" );
```

Comment:
Hooray ! I think all of us, we have to keep in mind clearing all the custom tables we are creating during uninstall.

Nice!